Author: Bernhard M. Wiedemann <bwiedemann suse de>
Date: 2018-03-11

Use ChangeLog date instead of build date
in order to make builds reproducible.
See https://reproducible-builds.org/ for why this is good.

Index: pencil-2.0.21/build/properties.sh
===================================================================
--- pencil-2.0.21.orig/build/properties.sh
+++ pencil-2.0.21/build/properties.sh
@@ -2,7 +2,7 @@
 
 export NAME='Pencil Prototyping'
 export VERSION='2.0.21'
-export BUILD=`date +%Y%m%d`
+export BUILD=`date -u -r ../CHANGELOG.md +%Y%m%d`
 export AUTHOR='Pavan Rikhi(pavan.rikhi@gmail.com) and Contributors'
 export MAINTAINER='Pavan Rikhi <pavan.rikhi@gmail.com>'
 export XPI_NAME='Pencil-'$VERSION'-firefox.xpi'
